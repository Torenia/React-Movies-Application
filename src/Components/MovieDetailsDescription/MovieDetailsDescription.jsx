import styled from 'styled-components';

export const MovieDetailsDescription = styled.div`
  color: #fff;
  
  div:first-child {
    display: flex;
    align-items: center;
    margin: 0;
    
    span {
      color: #0beb0b;
      margin-left: 10px;
      padding: 5px;
      font-size: 20px;
      border: 1px solid #fff;
      border-radius: 50%;
    }
  }
  
  span {
    font-size: 14px;
  }
  
  div:nth-last-child(2) {
    display: flex;
    align-items: center;
    color: #f65261;
    margin: 10px 0;
    
    div {
      margin-right: 10px;
    }
  }
  
  p {
    font-size: 15px;
  }
`;
