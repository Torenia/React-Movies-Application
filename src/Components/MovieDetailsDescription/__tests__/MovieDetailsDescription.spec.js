import React from 'react';

import { MovieDetailsDescription } from '../MovieDetailsDescription';

describe('<MovieDetailsDescription />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieDetailsDescription />);
  });

  it('should render a movie details description', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
