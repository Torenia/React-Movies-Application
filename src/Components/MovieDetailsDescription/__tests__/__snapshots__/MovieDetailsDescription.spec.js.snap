// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieDetailsDescription /> should render a movie details description 1`] = `
.c0 {
  color: #fff;
}

.c0 div:first-child {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0;
}

.c0 div:first-child span {
  color: #0beb0b;
  margin-left: 10px;
  padding: 5px;
  font-size: 20px;
  border: 1px solid #fff;
  border-radius: 50%;
}

.c0 span {
  font-size: 14px;
}

.c0 div:nth-last-child(2) {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #f65261;
  margin: 10px 0;
}

.c0 div:nth-last-child(2) div {
  margin-right: 10px;
}

.c0 p {
  font-size: 15px;
}

<div
  className="c0"
/>
`;
